templates/minelev/samtale/hemmelig-nb.md

```diff
@@ -0,0 +1,32 @@
+---
+definition: notatmal
+language: nb
+watermark: {{#if preview}} true {{else}} false {{/if}}
+to: {{ school.name }}
+info:
+  sector: Seksjon for sektorutvikling
+  our-date: {{ isoDate created.timestamp }}
+  paragraph: Offl. § 13 jf. fvl. §13 (1)
+---
+
+# Brev må sendes til {{ student.name }}
+
+Det er i dag produsert et brev i MinElev til {{ student.name }} i klasse {{ student.classId }} fra {{ teacher.name }} fordi eleven har takket nei til elevsamtale.
+
+Eleven har hemmelig adresse, og har derfor ikke fått sitt brev i sin digitale postkasse.
+
+Administrasjonen på {{ school.name }} må sørge for å skrive ut brevet som bekrefter at eleven har takket nei til elevsamtale fra 360. Brevet har tittel «Tilbud om elevsamtale» i 360. Deretter skal brevet gis til {{ teacher.name }}. Det er {{ teacher.name }} som da er ansvarlig å distribuere brevet videre til eleven.
```

Comment:
Burdet det ikke vært en `for` i den siste setningen?

Det er {{ teacher.name }} som da er ansvarlig `for` å distribuere brevet videre til eleven.

Reply:
Fixed!